odoo/fields.py

```diff
@@ -2868,7 +2868,7 @@ def read(self, records):
 
         # group lines by inverse field (without prefetching other fields)
         group = defaultdict(list)
-        for line in lines.with_context(prefetch_fields=False):
+        for line in lines:
```

Comment:
this and the backport of the other PR needs to be validated with the orm team

Reply:
This change is not acceptable.  The `prefetch_fields=False` was done on purpose, because it gives the best tradeoff on average.